.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a
+# variety of Python versions. For more information see:
+# https://help.github.com/actions/language-and-framework-guides/
+# using-python-with-github-actions
+
+name: Pytest with Flake8
+
+on: [pull_request, push]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: ["3.8", "3.9"]
+        numpy_ver: [latest]
+        include:
+          - python-version: "3.7"
+            numpy_ver: "1.18"
+            os: "ubuntu-latest"
+
+    name: Python ${{ matrix.python-version }} on ${{ matrix.os }} with numpy ${{ matrix.numpy_ver }}
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install standard dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r test_requirements.txt
+        pip install -r requirements.txt
+
+    - name: Install NEP29 dependencies
+      if: ${{ matrix.numpy_ver != 'latest'}}
+      run: |
+        pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
+
+    - name: Set up pysat
+      run: |
+        mkdir pysatData
+        python -c "import pysat; pysat.params['data_dirs'] = 'pysatData'"
+
+    - name: Test PEP8 compliance
+      run: flake8 . --count --select=D,E,F,H,W --show-source --statistics
+
+    - name: Evaluate complexity
+      run: flake8 . --count --exit-zero --max-complexity=10 --statistics
+
+    - name: Run OMMBV setup.py
+      run: |
+        pip install -e ../OMMBV -t ../OMMBV
```

Comment:
not windows compatible

Reply:
I have yet to find a windows compatible command that really installs the package. It goes through the motions, compiles the Fortran, etc, however, I can't get pytest to use whatever is installed.

Reply:
I think that it's skipping this part on windows.  Github Actions will still try to load code locally if not installed.

Could probably set up os-dependent jobs here, one for unix, one for windows.  But we'd need to test it on a windows machine locally, since I'm not sure if it's as easy as swapping directories. I'd bump this down the line since this is a big pull.  Perhaps @jonathonmsmith can help?

Reply:
I agree that I need help from someone with a local windows machine. From the Actions tab windows does run the pip install stuff and it runs without error. I used to have python setup.py develop which I'll probably go back to.